File: __tests__/__fixtures__/output/java/nethttp/nested.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value"))
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());

File: src/targets/javascript/jquery/fixtures/nested.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value',
  method: 'GET',
  headers: {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});